client/service.go

```diff
@@ -132,16 +139,37 @@ func NewService(
 	clNode chainlink.Node,
 	runtimeConfig store.RuntimeConfig,
 ) *Service {
+	upkeepExecuter := keeper.NewNoOpUpkeepExecuter()
+	registrySynchronizer := keeper.NewNoOpRegistrySynchronizer()
+	if runtimeConfig.KeeperEthEndpoint != "" {
```

Comment:
We don't start anything without the endpoint. The default is to use a NoOp.

Reply:
I think I understand what you mean now... I allowed for omitting the env var in case node ops were going to run other non-keeper EIs. But since we're going to make this it's own fork, I think we can make the ENV VAR required.

Reply:
https://www.pivotaltracker.com/story/show/176791569